src/Compilers/CSharp/Test/Syntax/Parsing/TopLevelStatementsParsingTests.cs

```diff
@@ -1610,26 +1610,16 @@ public void NewKeyword()
         }
 
         [Fact]
-        public void TupleUnsupportedInUsingStatement()
+        public void TupleUnsupportedInUsingStatement_CSharp9()
         {
             var test = @"
 using VT2 = (int, int);
 ";
 
-            UsingTree(test,
-                // (2,13): error CS1001: Identifier expected
-                // using VT2 = (int, int);
-                Diagnostic(ErrorCode.ERR_IdentifierExpected, "(").WithLocation(2, 13),
-                // (2,13): error CS1002: ; expected
-                // using VT2 = (int, int);
-                Diagnostic(ErrorCode.ERR_SemicolonExpected, "(").WithLocation(2, 13),
-                // (2,14): error CS1525: Invalid expression term 'int'
-                // using VT2 = (int, int);
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "int").WithArguments("int").WithLocation(2, 14),
-                // (2,19): error CS1525: Invalid expression term 'int'
-                // using VT2 = (int, int);
-                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "int").WithArguments("int").WithLocation(2, 19)
-                );
+            var tree = UsingTree(test, CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp9));
+
+            var actualErrors = tree.GetDiagnostics();
+            actualErrors.Verify();
```

Comment:
Consider using `CreateCompilation(test).VerifyDiagnostics(...);` instead, here and in test below.

Reply:
i will.  though i think this line is good to validate that the *tree* is fine.

Reply:
Added compilation tests as well.